lib/medera/minion.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Medera.Minion do
+  @moduledoc """
+  A Medera Minion is a worker node available to the Medera server
+  """
+
+  alias Medera.Minion.Registry
+
+  @doc "Returns a list of minion nodes"
+  @spec list() :: [atom]
+  def list do
+    Registry.list_minions |> Enum.map(&:erlang.node/1)
+  end
+
+  @doc "Returns the configured master node"
+  @spec master_node :: atom
+  def master_node do
+    case Application.get_env(:medera, :master_node) do
+      name when is_binary(name) -> String.to_atom(name)
```

Comment:
This is one of those cases where you need `to_atom` since there's no reason the master node's name would exist as an atom before this is called.